### ISSUE TITLE
Latest version on Pypi

### ISSUE THREAD

**Issue by torzdf** [nicolargo/nvidia-ml-py3#3]
First up, thanks for this extremely useful Lib.

I was wondering if it would be possible to push the latest version with the check into System32 for Windows users to PyPi?

It seems that Windows is increasingly installing Nvidia drivers this way, and it would be extremely useful to be able to pull this functionality in with Pip.

Many thanks, and keep up the good work!

**Comment by kapsh** [comment]
In the meantime you can do something like 
```
pip install git+https://github.com/nicolargo/nvidia-ml-py3
```

**Comment by torzdf** [comment]
Yeah, this is what we have. The main problem is that the version number in this repo is the same as in PyPi, so it won't automatically update to this version of nvidia-ml-py3 if it already exists on their system (this is for a requirements.txt install). I have had to fork this repo to update the version number.

It also means it also re-downloads the git version every time people try to update to check the version number.

**Comment by stas00** [comment]
> In the meantime you can do `pip install git+https://github.com/nicolargo/nvidia-ml-py3`

This doesn't work well with other libraries that depend on this library.

**Comment by iam-Shashank** [comment]
Would be quite helpful if the pypi version was updated.

**Comment by kapsh** [comment]
I'd suggest to ping or email repository owner directly for that. Not everyone has github comment notifications enabled.

**Comment by stas00** [comment]
do you have his email address? github doesn't expose repo owner's email address.

**Comment by patrikhuber** [comment]
It looks like NVIDIA has also updated their bindings on PyPi `nvidia-ml-py` to version `10.418.84` in 2019 https://pypi.org/project/nvidia-ml-py/#history. `pip install nvidia-ml-py` will still give a weird `nvidia-ml-py-375.53.tar.gz` version that I can nowhere find on PyPi but `pip install nvidia-ml-py==10.418.84` does the trick for me. That version looks like it's working with Python 3 too.
Does anyone know the differences between the two, what are you all using?

Edit: This is highly related: https://github.com/gpuopenanalytics/pynvml/issues/2.

**Comment by wookayin** [comment]
As of 2021, there is no need to use this library or `nvidia-ml-py3` package -- **PLEASE DON'T USE IT!**

NVIDIA is maintaining an official python binding (which also is compatible with python3): https://pypi.org/project/nvidia-ml-py/

FYI as of today the latest version is [11.460.79](https://pypi.org/project/nvidia-ml-py/#history) which is released on May 2021, including recent changes to the NVML API.

Regarding Patrik's comment, there is a bit hiccup in package versions, see also: https://forums.developer.nvidia.com/t/mistake-with-nvidia-ml-py-on-pypi/112942 for more details. NVIDIA will remove the 375.xx package soon. For now, you can do:

```
pip install nvidia-ml-py!=375.*
```

@nicolargo Could you please consider informing this information in the repository?
